fiddler/datadog_checks/fiddler/check.py

```diff
@@ -169,10 +169,27 @@ def _process_v1compat_query_result(
         value_list = result['data'][-1] if result['data'] else 0
         gauge_results = []
 
-        # Update metric names if necessary
-        metric_mapping = {'traffic': 'traffic_count', 'jsd': 'histogram_drift'}
+        # metric name mapping from 23.7 -> 23.5 format
+        metric_mapping = {
+            'traffic': 'traffic_count',
+            'jsd': 'histogram_drift',
+            'expected_calibration_error': 'expected_callibration_error',
```

Comment:
Are we purposely renaming this metric to a typo? Or is `callibration` a specific term for used fiddler?

Reply:
This is a typo in the older version of Fiddler which has been fixed.
This map ensures backwards compatibility with older Dashboards that customers are on.

Reply:
I guess would it be better to update to fix the typo in the new version and then sync the dashboard to use the corrected version? @emarsha94 what do you think? I'm okay with merging this as is.

Reply:
To confirm, the metric has a typo so we updated the metric to fix the type. However, the old metric is still being used? If it's not used moving forward then we should update the dashboard with the new version. Customers can pull previous metric history on the old metric as needed. You can include two metrics in a graph widget if you want customers to see both.

Reply:
Rechecked dashboard usage. There is no consumer of this metric.
The default dashboard available on Datadog does not use this metric either.

I think, we should be fine.

Reply:
@emarsha94 Yeah basically that. Older version of the fiddler has a typo in the metric name and the new version of fiddler fixed it. The line I'm commenting on remaps the corrected metric name to the older typo version.

I'll approve this PR as this is more of a design choice than a coding issue at this point. I'll go with whatever you both decide on.

Reply:
Nope. The default dashboard does not use it.